src/Containers.jsx

```diff
@@ -442,6 +446,102 @@ class Containers extends React.Component {
         this.setState({ width: this.cardRef.current.clientWidth });
     }
 
+    podStats(pod) {
+        const { containersStats } = this.props;
+        // when no containers exists pod.Containers is null
+        if (!containersStats || !pod.Containers) {
+            return null;
+        }
+
+        // As podman does not provide per pod memory/cpu statistics we do the following:
+        // - don't add up CPU usage, instead display the highest found CPU usage of the containers in a pod
+        // - add up memory usage so it displays the total memory of the pod.
+        let cpu = 0;
+        let mem = 0;
+        for (const container of pod.Containers) {
+            const containerStats = containersStats[container.Id + pod.isSystem.toString()];
+            if (!containerStats)
+                continue;
+
+            if (containerStats.CPU != undefined) {
+                const val = containerStats.CPU === 0 ? containerStats.CPU : containerStats.CPU.toFixed(2);
+                if (val > cpu)
+                    cpu = val;
+            }
+            if (containerStats.MemUsage != undefined)
+                mem += containerStats.MemUsage;
+        }
+
+        return {
+            cpu,
+            mem,
+        };
+    }
+
+    renderPodDetails(pod) {
+        const podStats = this.podStats(pod);
+        const infraContainer = this.props.containers[pod.InfraId + pod.isSystem.toString()];
+        const infraContainerDetails = this.props.containersDetails[pod.InfraId + pod.isSystem.toString()];
+
+        // Wait till we have valid data before rendering
```

Comment:
Even when running it might consume none resources for time being. I would say it should always be present when the pod is 'running' and just show `0`. See the video (testing this new feature integrated into gnome :) )

[Screencast from 2022-09-19 09-36-55.webm](https://user-images.githubusercontent.com/12330670/190970381-df8ba9a6-d81a-4948-a7b2-5a971305cc24.webm)

Reply:
Also I cannot inspect open ports when containers are not busy enough

Reply:
You should probably be able to see container ports/mounts regardless.